plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -413,6 +413,7 @@ def updateGUIFromParameterNode(self, caller=None, event=None):
         self.ui.inputSelector.clear()
         for v in self._volumeNodes:
             self.ui.inputSelector.addItem(v.GetName())
+            self.ui.inputSelector.setToolTip(self.current_sample["name"])
```

Comment:
will be better if we have condition check here..
`self.current_sample.get("name", "") if self.current_sample else ""`

Reply:
there can be a case where next sample is not fetched.. and user can load his/her own image in slicer and continue to work as temp session... in that case.. i believe current_sample will be empty.. so i think it's better to have this check.. as we allow users to bring his/her own images

Reply:
changed accordingly :-)